tests/unit/test_absolute_freq_ssb.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from src.du_parameters.absolute_freq_ssb import (
+    AbsoluteFrequencySSBError,
+    get_absolute_frequency_ssb,
+)
+from src.du_parameters.frequency import ARFCN, GSCN, Frequency
+
+
+class TestAbsoluteFrequencySSB:
+    def test_get_absolute_frequency_ssb_when_valid_input_given_then_return_expected_result(self):
+        mock_center_freq = MagicMock(spec=Frequency)
+        mock_gscn = MagicMock(spec=GSCN)
+        mock_adjusted_frequency = MagicMock(spec=Frequency)
+        mock_arfcn = MagicMock(spec=ARFCN)
+
+        with (
+            patch.object(GSCN, "from_frequency", return_value=mock_gscn) as mock_from_frequency,
+            patch.object(
+                GSCN, "to_frequency", return_value=mock_adjusted_frequency
+            ) as mock_to_frequency,
+            patch.object(
+                ARFCN, "from_frequency", return_value=mock_arfcn
+            ) as mock_from_frequency_arfcn,
+        ):
+            result = get_absolute_frequency_ssb(mock_center_freq)
+            assert result == mock_arfcn
+            mock_from_frequency.assert_called_once_with(mock_center_freq)
+            mock_to_frequency.assert_called_once_with(mock_gscn)
+            mock_from_frequency_arfcn.assert_called_once_with(mock_adjusted_frequency)
+
```

Comment:
```suggestion
```

I would personally remove this test, the only thing it is really testing is the implementation details. This will only make the code harder to change.